plugins/experiment_procedure.cc

```diff
@@ -407,7 +407,7 @@ run(const ProcedureRunConfig & run,
         if(runProcConf.outputAccuracyDataset) {
             PolyConfigT<Dataset> outputPC;
             outputPC.id = ML::format("%s_results_%d", runProcConf.experimentName, (int)progress);
-            outputPC.type = "sparse.mutable";
```

Comment:
But the output dataset in classifier is an optional, do it doesnt have a default... @nicolaskruchten ?